pkg/crc/ssh/client.go

```diff
@@ -105,6 +106,10 @@ func (client *NativeClient) Run(command string) ([]byte, []byte, error) {
 	session.Stderr = &stderr
 
 	err = session.Run(command)
+	if err != nil {
+		client.stale = true
+		client.conn.Close()
```

Comment:
I am curious: what happens for instance when crc start polls for the pull secret file on disk. Does it reconnect ssh for each try?

Reply:
@guillaumerose with this patch yer, but I am looking into specific error to filter this out so that we only perform this action when connection reset happen.